src/poly/commitment.rs

```diff
@@ -14,8 +15,20 @@ pub trait PolynomialCommitmentScheme<F: PrimeField>: Clone + Debug {
     /// Parameters needed to verify a proof in the PCS
     type VerifierParameters;
 
+    /// Output of scalar multiplication for a Self::Commitment
+    type Output: Into<Self::Commitment>;
     /// Type of a committed polynomial
-    type Commitment: Clone + Copy + Debug + Default + PartialEq + ProcessedSerdeObject + Send + Sync;
+    type Commitment: Clone
+        + Copy
+        + Default
+        + Debug
+        + Default
+        + PartialEq
+        + ProcessedSerdeObject
+        + Send
+        + Sync
+        + Add<Output = Self::Output>
+        + Mul<F, Output = Self::Output>;
```

Comment:
I guess you could have created a homomorphic trait here instead :D if you wanted to keep it somewhat distinct for what you had in mind